Purchases/Public/Purchases.swift

```diff
@@ -993,15 +993,11 @@ public extension Purchases {
      */
     @available(iOS 12.2, macOS 10.14.4, watchOS 6.2, macCatalyst 13.0, tvOS 12.2, *)
     @objc(purchasePackage:withDiscount:completion:)
-    func purchase(package: Package, discount: SKPaymentDiscount, completion: @escaping PurchaseCompletedBlock) {
-        // todo: add support for SK2 with discounts, move to new class
-        // https://github.com/RevenueCat/purchases-ios/issues/1055
```

Comment:
I think this comment is still valid - ideally `Purchases.swift` is just a facade and actual decision-making regarding sk1 vs sk2 happens somewhere else

Reply:
The `purchase` method just delegate to `PurchasesOrchestrator` now though? I do agree there's lots of logic in this class that can be refactored out, but I don't think I see this comment being relevant anymore?